lib/dpl/provider/chef_supermarket.rb

```diff
@@ -33,7 +37,7 @@ def check_app
         # So we assume cookbook path is '..'
         cl = ::Chef::CookbookLoader.new '..'
         @cookbook = cl[cookbook_name]
-        ::Chef::CookbookUploader.new(cookbook, '..').validate_cookbooks
+        cl.validate_cookbooks
```

Comment:
In looking at the `Chef::CookbookLoader` object, I'm not seeing `validate_cookbooks' but I do see something that might be similar to it.

https://github.com/chef/chef/blob/master/lib/chef/cookbook_loader.rb#L113

`cookbook_exists?`

However, digging a bit deeper.  It looks like, `Chef::CookbookUploader` contains the method `validate_cookbooks`.

https://github.com/chef/chef/blob/master/lib/chef/cookbook_uploader.rb#L140

Reply:
Perhaps, this might be alright?

`::Chef::CookbookUploader.new(cookbook).validate_cookbooks`

Reply:
Let's try that.